internal/sddc/sddc_manager_subresource.go

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2023 VMware, Inc.
+ *    SPDX-License-Identifier: MPL-2.0
+ */
+
+package sddc
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	utils "github.com/vmware/terraform-provider-vcf/internal/resource_utils"
+	validation_utils "github.com/vmware/terraform-provider-vcf/internal/validation"
+	"github.com/vmware/vcf-sdk-go/models"
+)
+
+func GetSddcManagerSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		MaxItems: 1,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"hostname": {
+					Type:         schema.TypeString,
+					Description:  "SDDC Manager Hostname, length 3-63",
```

Comment:
hostname or fqdn?

Reply:
both are supported - if just the short hostname is provided, then FQDN will be generated using the subdomain from dnsSpec